CMakeLists.txt

```diff
@@ -65,11 +65,42 @@ endif()
 target_include_directories(nanothread
   PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+  $<INSTALL_INTERFACE:include>)
```

Comment:
Not sure I understand this change. Can you explain what's wrong with `CMAKE_INSTALL_INCLUDEDIR?`

Reply:
I actually forget at this point, but I think the former expands into an absolute path specific to the build machine, https://cmake.org/cmake/help/latest/guide/importing-exporting/index.html#creating-relocatable-packages